src/Conf.ml

```diff
@@ -296,8 +296,10 @@ let info =
          the processed file."
     ; `P
         "(*) .ocamlformat files in current and all ancestor directories \
-         for each input file are used, with more distant ancestors having \
-         lower priority." ]
+         for each input file are used, as well as the global .ocamlformat \
+         file defined in $(b,\\$XDG_CONFIG_HOME/ocamlformat). The global \
```

Comment:
filename out of sync with code

Reply:
I suppose the global .ocamlformat file is placed in the directory $XDG_CONFIG_HOME/ocamlformat so the doc looks ok to me. Do you want to rename it ?

Reply:
Ok, I thought the doc was giving the filename rather than the directory name. Either is fine, but my impression is that files under $XDG_CONFIG_HOME are often named differently than when in $HOME, in particular without a leading `.`. I don't have any strong feeling, just think we should follow convention.